src/github_changelog/github.clj

```diff
@@ -15,13 +15,16 @@
                   :or   {github-api (:github-api defaults/config)}}]
   (format "%s/repos/%s/%s/pulls" (util/strip-trailing github-api) user repo))
 
+(defn- auth-headers [token]
```

Comment:
I am not in favor of private defns anymore as a concept, I think it would be fine dropping them (even adjusting the rest of the fns in this ns).

Reply:
I made all the functions public that are touched in this change.